### DISCUSSION TITLE
Allow pcp to do multi-colour plots like scatter

### DISCUSSION THREAD

**Discussion by BCheong1990** [anyoptimization/pymoo#251] [Ideas]
Hi Julian,

Can I suggest that for the pcp file (lib\site-packages\pymoo\visualization\pcp.py), you change line 78 to 'set_if_none(_kwargs, "color", self.colors[k % len(self.colors)])', similar to that of the scatter plot?

That will allow multi-colour plots to be produced!

Many thanks,
Benjamin

**Comment by blankjul** [comment]
Hi Benjamin,

good catch! It will be fixed in the new version. Thanks for pointing it out.
If there is anything else or if you have any other questions, please let me know.

Julian
